pkg/reconciler/service/workerpool.go

```diff
@@ -179,3 +100,7 @@ func (wa *WorkerPool) IsClosed() bool {
 	}
 	return wa.antsPool.IsClosed()
 }
+
+func (wa *WorkerPool) RunningWorkers() int {
+	return wa.antsPool.Running()
+}
```

Comment:
The worker pool size could be retrieved by the occupancyTracker by calling another function like `func (wa *WorkerPool) Size() int {...}`